src/applications/mhv-secure-messaging/components/ComposeForm/ComposeForm.jsx

```diff
@@ -333,13 +337,17 @@ const ComposeForm = props => {
         setCategoryError(ErrorMessages.ComposeForm.CATEGORY_REQUIRED);
         messageValid = false;
       }
-      if (!isDraft && isSignatureRequired && !digitalSignature) {
+      if (!isDraft && isSignatureRequired && !electronicSignature) {
         setSignatureError(ErrorMessages.ComposeForm.SIGNATURE_REQUIRED);
         messageValid = false;
       }
       if (signatureError !== '') {
         messageValid = false;
       }
+      if (!checkboxMarked) {
```

Comment:
I think this needs to be corrected. We need to identify message as invalid due to unchecked checkbox only if such is needed. Otherwise, other messages cannot be sent too
```suggestion
      if (!isDraft && isSignatureRequired & !checkboxMarked) {
```